simple_qadi_demo.py

```
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+"""
+Simple QADI cycle demonstration script.
+"""
+
+import asyncio
+from mad_spark_alt.core import SmartQADIOrchestrator
+
+async def run_qadi_demo():
```

Ruff (0.11.9)

Missing return type annotation for public function `run_qadi_demo`

Add return type annotation: `None`

(ANN201)